stream-compositions/pom.xml

```diff
@@ -27,14 +27,14 @@
     </modules>
 
     <properties>
-        <backbase-bom.version>2023.12</backbase-bom.version>
+        <backbase-bom.version>2024.04</backbase-bom.version>
```

Comment:
if we are using the 2023.03-LTS version in the main pom xml we should keep this version here consistent

Reply:
done